.pipelines/cnm-image.yml

```diff
@@ -0,0 +1,25 @@
+steps:
+  - task: GoTool@0
+    inputs:
+      version: '1.18.5'
+  - bash: |
+      git show --stat
+      echo $REGISTRY_PASSWORD | docker login $REGISTRY_URL -u $REGISTRY_USERNAME --password-stdin
+      export IMAGE_REGISTRY=$REGISTRY_URL
+      IMAGE_NAME="azure-cloud-controller-manager"
+      IMAGE_TAG="$(git rev-parse --short=7 HEAD)"
```

Comment:
Maybe variables like IMAGE_NAME and IMAGE_TAG here can be removed?

Reply:
Yes. Removed.